src/action.js

```diff
@@ -105,9 +109,25 @@ function parsePrTitle(pullRequest) {
   if (!match) {
     throw new Error('Error while parsing PR title, expected: `bump <package> from <old-version> to <new-version>`')
   }
-  const [, oldVersion, newVersion] = match
+
+  const [, oldVersion, newVersion] = match.map(t => t.replace(/`/g, ''))
 
   const packageName = getPackageName(pullRequest.head.ref)
 
-  return { [packageName]: { delete: semverCoerce(oldVersion).raw, insert: semverCoerce(newVersion).raw } }
+  if (semverCoerce(oldVersion) && semverCoerce(newVersion)) {
```

Comment:
there's a better way to write this whole section using conditional chaining, let's leverage that

Reply:
I guess it could be improved, but not by just doing something like this (as it's what I'm assuming you mean:
```js
return {
  [packageName]: {
    delete: semverCoerce(oldVersion)?.raw ?? oldVersion,
    insert: semverCoerce(newVersion)?.raw ?? newVersion
  }
}
```
I just noticed that `semver.coerce` sometimes will be a bit too loose when handling some inputs which caused some of the tests to ffail, for instance it will parse cc**221**b3 as **221**.0.0 and such. I'll use a ternary based on `semver.valid` which will reduce the code repetition.

I'm just fixing the code coverage that was reduced after this change.